src/java/com/zimbra/cs/nginx/NginxLookupExtension.java

```diff
@@ -889,19 +889,17 @@ private DomainExternalRouteInfo getDomainExternalRouteInfo(ILdapContext zlc, Con
          */
         private Pair<Server, Boolean> lookupUpstreamImapServer(NginxLookupRequest req) throws ServiceException {
             ImapLoadBalancingMechanism LBMech = ImapLoadBalancingMechanism.newInstance();
-            HashMap<String, Boolean> servers = new HashMap<>();
+            boolean usingRemoteImapDaemon = true /* initial assumption */;
```

Comment:
nit: I'm not sure how others feel, but I prefer the semicolon to follow the ```true``` and then have the comment follow

Reply:
I completely overlooked this myself.

Reply:
@plobbes I prefer that too.  Will make that change along with some PMD fixes.